build-configuration/pom.xml

```diff
@@ -125,6 +125,7 @@
       <version.cdi>2.0</version.cdi>
       <version.codehaus.jackson>1.9.13</version.codehaus.jackson>
       <version.console>0.1.Final</version.console>
+      <version.datahoud>1.0.1.RELEASE</version.datahoud>
```

Comment:
Is this meant to be `version.blockhound`?

Reply:
Whoops, yeah.